### ISSUE TITLE
update to Waterdist() possibly needed

### ISSUE THREAD

**Issue by iep-coder** [sbashevkin/spacetools#2]
When using the Waterdist() function today, I received the following warning: 

"as(<dsCMatrix>, "dgTMatrix") is deprecated since Matrix 1.5-0; do as(as(., "generalMatrix"), "TsparseMatrix") instead"

At least for now, the function still works.

**Comment by sbashevkin** [comment]
Yeah I used it recently and noticed that warning too. Unfortunately I think that warning comes from a dependency since I can't find `as(, "dgTMatrix")` anywhere in `spacetools`. But it still worked for me as well.
